locales/pt-BR/translation.json

```
@@ -96,6 +96,43 @@
   "$FixedDayCalendar": {
     "leave-by": "Você deve sair às:"
   },
+  "$WorkdayWaiver": {
+    "add-waiver": "Adicionar renúncia",
+    "already-have-waiver": "Você já tem uma renúncia em",
+    "attention": "Atenção: A terceirização de feriados em dias com isenção anulará a isenção.",
```

I think a better translation would be:

```suggestion
    "add-waiver": "Adicionar folga",
    "already-have-waiver": "Você já tem uma folga em",
    "attention": "Atenção: O carregamento de feriados em dias com folga sobreescreverá a entrada.",
```